### PR TITLE
add branches allowed

### PR DESCRIPTION
**Pull request by vonglasow**
add allowed branches in travis for born

### CODE REVIEW

File: bors.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+status = [
+  "continuous-integration/travis-ci/pr"
```
Comment (notriddle) [inline bors.toml]: This line is not what you want.

What you probably want is `"continuous-integration/travis-ci/push"`, [like this](https://github.com/nix-rust/nix/blob/master/bors.toml#L2).

**Comment by vonglasow** [comment]
bors delegate+

File: bors.toml (hunk 1 above)
Comment (vonglasow) [reply bors.toml]: thanks I have tried but it's still timeout and everything is configured as explain in the FAQ.

Comment (notriddle) [reply bors.toml]: I'm not sure why everything is timing out, but I can still tell you that it needs to be the Push one, not the PR one. The PR one will not work.

**Comment by notriddle** [comment]
[Here's the build](https://travis-ci.org/vonglasow/rust-of-life/builds/270197229).

**Comment by vonglasow** [comment]
@notriddle yes and it works now don't know what I missed the only thing I have added bors add reviewer on the interface. thanks for your help

**Comment by notriddle** [comment]
Adding bors as a reviewer is *not* it.

**Pull request merged**